cluster/images/hyperkube/Makefile

```diff
@@ -33,7 +37,9 @@ ifndef VERSION
     $(error VERSION is undefined)
 endif
 	cp -r ./* ${TEMP_DIR}
-	cp ${HYPERKUBE_BIN} ${TEMP_DIR}
+
+	tar -cvzf ${TEMP_DIR}/binaries.tgz  -C ${OUTPUT_PATH} kube-apiserver kube-controller-manager \
+		kube-proxy kube-scheduler kubectl kubelet
 
 	chmod a+rx ${TEMP_DIR}/hyperkube
```

Comment:
is this still needed? do we need to do the same for the binaries we extracted?

Reply:
the binaries are being zipped up here, the zip has the correct permission bits. so we should be fine. the +x on the script is insurance i think (from before)

Reply:
yeah, to avoid umask issues when building